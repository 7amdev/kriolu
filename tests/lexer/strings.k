"2"  "Hello World."
"some random 
string with break line and 
numbers 1234"
"      "
"Hello "World"""!"