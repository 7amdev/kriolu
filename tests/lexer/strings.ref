 1 <string>  '""' 
 2 <string>  '"2"' 
 2 <string>  '"Hello World."' 
 6 <string>  '"some random 
string with break line and 
numbers 1234"' 
 7 <string>  '"      "' 
 8 <string>      '"Hello "' 
 8 <identifier>  'World' 
 8 <string>      '""' 
 8 <string>      '"!"' 