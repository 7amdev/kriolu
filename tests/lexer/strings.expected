1 <string>                 '"2"' 
1 <string>                 '"Hello World."' 
4 <string>                 '"some random 
string with break line and 
numbers 1234"' 
5 <string>                 '"      "' 
6 <string>                 '"Hello "' 
6 <identifier>             'World' 
6 <string>                 '""' 
6 <string>                 '"!"' 